index.js

```diff
@@ -1,20 +1,21 @@
-let express = require('express');
-let cors = require('cors');
```

Comment:
Doesn't the cors middleware add the options route? I assume through this tutorial the smart sandbox will call this server running on localhost. If thats the case I don't _think_ we need to worry about cors, but we should verify. If this server was deployed cors and the options route would have to be setup.

If we don't need cors for this tutorial, I'd remove it entirely to simplify the server.